themes/bootstrap3/templates/Recommend/SideFacets/range-slider.phtml

```diff
@@ -1,8 +1,12 @@
 <?php
   // Sanitize inputs:
-  $sanitize = function ($val) {
-    return preg_replace('/[^\d]*/', '', $val);
-  };
+  $sanitize = 'fulldate' === $this->facet['type']
```

Comment:
Is 'fulldate' facet type something we actually support, or is this something local on your end? Doesn't look familiar to me. Certainly worth supporting, just wondering if there are more pieces to this that we need to add, or if I'm just missing something.

Reply:
It is (see SideFacets::getAllRangeFacets). Not sure if it's actually used, but it could be.

Reply:
Ahh, I see, it's the `fullDateRange` setting in facets.ini. Somehow I couldn't find that the first time I looked.